static/src/javascripts/bootstraps/standard/alreadyVisited.ts

```diff
@@ -0,0 +1,24 @@
+import { onConsent } from '@guardian/consent-management-platform';
```

Comment:
duplicated from the DCR code - https://github.com/guardian/dotcom-rendering/pull/9857